SharedContent/cs-winui-template/WinUI3TemplateCs/MainWindow.xaml.cs

```diff
@@ -0,0 +1,110 @@
+﻿using Microsoft.UI.Xaml;
+using System;
+using System.Runtime.InteropServices;
+using WinRT;
+using Microsoft.Win32.SafeHandles;
+using Windows.Win32.Foundation;
+using Windows.Win32.UI.Controls;
+using Windows.Win32.UI.WindowsAndMessaging;
+using Windows.Win32.Graphics.Gdi;
+using static Windows.Win32.PInvoke;
+using static Windows.Win32.Constants;
+
+namespace WinUI3TemplateCs
+{
+    public sealed partial class MainWindow : Window
+    {
+        public MainWindow()
+        {
+            this.InitializeComponent();
+
+            Title = Settings.FeatureName;
+
+            HWND hwnd = (HWND)WinRT.Interop.WindowNative.GetWindowHandle(this);
+            LoadIcon(hwnd,"Assets/windows-sdk.ico");
+            SetWindowSize(hwnd, 1220, 1080);
+            PlacementCenterWindowInMonitorWin32(hwnd);
+        }
+
+        private unsafe void LoadIcon(HWND hwnd, string iconName)
+        {
+            const int ICON_SMALL = 0;
+            const int ICON_BIG = 1;
+
+            fixed (char* nameLocal = iconName)
+            {
+                HINSTANCE hInstLocal = default(HINSTANCE);
```

Comment:
No need for a local declaration here. You can just use `default(HINSTANCE)` as the first argument.